napari_clusters_plotter/_Qt_code.py

```diff
@@ -67,6 +67,9 @@ def layer_container_and_selection():
     layer_select = create_widget(annotation=Layer, label="layer")
     layer_selection_container.layout().addWidget(layer_select.native)
 
+    if viewer is not None and viewer.layers.selection.active is not None:
+        layer_select.value = viewer.layers.selection.active
```

Comment:
I'm speculating this crashes when two layers are selected in napari?

Reply:
I tested this and at the moment when you select two layers the selection.active returns none. I’m guessing that might change some time in the future though but I’m not sure